js/plugins/dotprompt/src/prompt.ts

```diff
@@ -42,11 +42,11 @@ import { compile } from './template.js';
 
 export type PromptData = PromptFrontmatter & { template: string };
 
-export type PromptGenerateOptions<V = unknown> = Omit<
-  GenerateOptions<z.ZodTypeAny, typeof GenerationCommonConfigSchema>,
-  'prompt' | 'model'
-> & {
-  model?: string;
+export type PromptGenerateOptions<
+  V = unknown,
```

Comment:
Any reason not to change `V` to `I` like we have everywhere else? `V` doesn't really make me think "input".

Reply:
That works.

Reply:
Changed `V` and `Variables` to `I` to be consistent with the rest of the codebase.